content/fundamentals/unit-testing.md

```diff
@@ -198,6 +198,8 @@ describe('CatsController', () => {
 
 You can also retrieve these mocks out of the testing container as you normally would custom providers, `moduleRef.get(CatsService)`.
 
+Do note that `REQUEST` and `TRANSIENT` providers cannot be auto-mocked because they're already defined in the context, but they can be _overwritten_ with a custom provider or a `.overrideProvider`.
```

Comment:
These are specifically the `REQUEST` and `INQUIRER`providers, right? I may have misspoken with the use of `TRANSIENT` in the related issue. I believe that providers that are scoped to `REQUEST` and `TRANSIENT` are fine, but the `REQUEST` and `INQUIRER` providers specifically cannot be auto-mocked